pyqt5-gui/about.py

```diff
@@ -0,0 +1,84 @@
+import sys
+
+from PyQt5.QtWidgets import (
+    QVBoxLayout,
+    QHBoxLayout,
+    QPushButton,
+    QLabel,
+    QWidget,
+    QApplication
+)
+from PyQt5.QtGui import *
+from PyQt5.QtCore import *
+
+
+class About(QWidget):
+    def __init__(self):
+        super().__init__()
+        self.setWindowTitle("About Us")
+        self.setGeometry(650, 300, 600, 400)
+
```

Comment:
Import customer font using the following block of code

```python
        font = QFontDatabase.addApplicationFont(r'../fonts/jetbrains-mono.regular.ttf')
        if font >= 0:
            fonts = QFontDatabase.applicationFontFamilies(font)
        else:
            print("Error loading fonts!")
```

Then, you can set the customer font to any object below, e.g.

```python
        contact_us = QLabel()
        contact_us.setText("<b><font size='4'>Contact Us</font></b><br>"
                           "123-456-7890<br>"
                           "stake_robe@example.com")
        contact_us.setFont(QFont(fonts[0], 12)) # where 12 is the font size
```